baselines/polars/README.md

```diff
@@ -0,0 +1,19 @@
+# Setup
+
+1. You need to drag the data folder (e.g. `tables-scale_1`) into the `tpch` dir.
```

Comment:
Ideally we want it to be able to use a pre-defined data directory. In our case, `$ROOT_DIR/resources/tpc-h/data/scale=1/partition=1/*.tbl` as we won't be creating copies of the data.

Reply:
I set the default data path to this pre-defined one.